kernel/kernel-api/src/main/java/io/delta/kernel/internal/stats/FileSizeHistogram.java

```diff
@@ -54,6 +54,25 @@ public static FileSizeHistogram createDefaultHistogram() {
     return new FileSizeHistogram(defaultBoundaries, zeroCounts, zeroBytes);
   }
 
+  /**
+   * Creates a FileSizeHistogram from a Row object.
+   * This method converts a Row object into a FileSizeHistogram instance. The Row must conform
+   * to the FULL_SCHEMA defined for FileSizeHistogram.
+   *
+   * @param row The Row object containing histogram data. Must not be null and must match FULL_SCHEMA.
+   * @return A new FileSizeHistogram instance created from the provided Row.
+   * @throws NullPointerException if the row is null
+   * @throws IllegalArgumentException if the row's schema doesn't match FULL_SCHEMA
+   **/
+  public static FileSizeHistogram fromRow(Row row) {
```

Comment:
@huan233usc actually is this only used once in tests or is it used more in future PRs? we could just put this hack to use `fromColumnVector` in the test code in that case